org.opentransactions.metier.json

```diff
@@ -0,0 +1,215 @@
+{
+  "id": "org.opentransactions.metier",
+  "runtime": "org.kde.Platform",
+  "runtime-version": "5.15",
+  "runtime-commit": "3522f785effad84e56b127dc800e2982ae4eeca621d7b12a1a26136039d452ae",
```

Comment:
```suggestion
```

Reply:
It's strange that I don't remember adding that line when I wrote the manifest. Did flatpak-builder do that?